### PR TITLE
[pvr] Fix wrong notifications when stopping/deleting recordings/timers

### PR DESCRIPTION
**Pull request by Jalle19**
#### Scenario 1:
- start a recording
- stop that recording

What happens: a "recording deleted" message is shown in almost all cases, when in fact the recording was never deleted, just stopped. This fixes that. The problem is that we've been looking at the current time and the end time for the recording when determining what message to display. I've refactored this to look at the last known state of the timer, which is both simpler and more reliable.
#### Scenario 2:
- schedule a recording
- delete the schedule

What happens: a "recording deleted" message is shown, when in fact there was no recording to begin with, just a timer. This fixes that too by simply changing the message string (it's not used anywhere else). I left this as a separate commit so we can drop it if you feel it's too late for language changes at this stage.

@xhaggi @ksooo you okay with this?

### CODE REVIEW

**Comment by Jalle19** [comment]
@opdenkamp pinging you too

**Comment by da-anda** [comment]
When a client is running and the backend f.e. is rebooting, on reconnection I often got "recording deleted" or "timer deleted" notifications which are a bit shocking at first. Will this also be fixed with this PR?

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -8603,7 +8603,7 @@ msgid "Recording completed"
 msgstr ""
 
 msgctxt "#19228"
-msgid "Recording deleted"
+msgid "Timer deleted"
```
Comment (da-anda) [inline addons/resource.language.en_gb/resources/strings.po]: mind adding a comment where this label is used as you already did the research :) Also a contextual comment would be great, something like
#. notification after a PVR timer was deleted

**Comment by Jalle19** [comment]
@da-anda I'm not sure your specific issue will be fixed, I'd need to see a debug log to know what's going on.

**Comment by ksooo** [comment]
What message now appears when you actually delete a recording from the recordings window?

**Comment by da-anda** [comment]
@Jalle19 I'll try to get a debug log

**Comment by Jalle19** [comment]
@ksooo nothing, nothing ever did.

**Comment by Jalle19** [comment]
We don't show anything when a movie is deleted either so it's not a bug.

**Comment by ksooo** [comment]
@Jalle19 ah, okay. Is this also true if a recording gets deleted using the pvr backend UI?

What about PVR_TIMER_STATE_COMPLETED? Can it happen that we run in GetDeletedNotifcationText in this state? If so, should it return "recording deleted" in this case? Not sure about this.

**Comment by Jalle19** [comment]
During my testing I never encountered that, but it's easy to add another case.

**Comment by ksooo** [comment]
Despite from my questions, looking good. +1 from my side. Another ancient problem solved. Yes!

**Comment by Jalle19** [comment]
If no one has any objections I'll merge this later today.

**Comment by Jalle19** [comment]
jenkins build this please

**Comment by opdenkamp** [comment]
+1, looks like I forgot this one when adding the time states

**Comment by Jalle19** [comment]
Build failure unrelated

**Pull request merged**